asset-metadata/app.yaml

```diff
@@ -0,0 +1,10 @@
+---
+runtime: python39
```

Comment:
Image we use is python3.10 - is it ok to run on different runtime?

Reply:
3.9 is the latest supported by app engine.